website/src/app/core/state/page/page.state.ts

```diff
@@ -19,8 +21,16 @@ interface PageStateModel {
 export class PageState extends NgxsImmutableDataRepository<PageStateModel> {
   readonly hasShownHelpModal$ = this.state$.pipe(pluck('hasShownHelpModal'));
 
+  ngxsOnInit(): void {
+    super.ngxsOnInit();
+    this.patchState({
+      hasShownHelpModal: localStorage.getItem(LOCAL_STORAGE_HELP_MODAL_KEY)?.toLowerCase() === 'true' ? true : false
```

Comment:
This should probably move to defaults in the @State decorator (line 17) and could be simplified to just `localStorage.getItem(LOCAL_STORAGE_HELP_MODAL_KEY)?.toLowerCase() === 'true'`

Reply:
I disagree. I think defaults in the @State decorator should be kept as constants and not execute a bunch of code. Also if we ever were to change from where the value is loaded it is less likely to cause problems.

Reply:
Fair point, what about as a constant above in the file. I don't like the patchState piece.

Reply:
`localStorage.getItem(...)` is dynamic and should preferably be executed at the time the hasShownHelpModal value is set. There is some cleanup that can happen though. Ex:
```
ngxsOnInit(): void {
  super.ngxsOnInit();
  const hasShownHelpModal = localStorage.getItem(LOCAL_STORAGE_HELP_MODAL_KEY)?.toLowerCase() === 'true'; // Ternary operator not needed
  this.patchState({ hasShownHelpModal });
}
```

Reply:
Note: Using `ngxsOnInit` and `patchState` is the correct way to load and set dynamic values during state initialization

Reply:
Ok

Reply:
@adphillips2017 See above for changes